main.go

```diff
@@ -271,6 +278,25 @@ func ShowVersion() {
 	os.Stdout.Write([]byte(v))
 }
 
+func PrintArtifactName() {
+	aname, err := ioutil.ReadFile("/etc/mender/artifact_info")
+	if err != nil {
+		fmt.Printf("failed to read artifact name: err: %v", err)
+		return
+	}
+	s := bufio.NewScanner(bytes.NewBuffer(aname))
+	for s.Scan() {
+		if strings.HasPrefix(s.Text(), "artifact_name=") {
+			a := strings.Split(s.Text(), "=")
+			if len(a) == 2 {
+				fmt.Println(a[1])
+			} else {
+				fmt.Println("Wrong formatting of the artifact_info file")
+			}
+		}
+	}
```

Comment:
There are a couple of corner cases here that I think there should produce an error:

* There is no `artifact_name=`
* There is more than one `artifact_name=`

Both can have severe consequences if they are wrong, so I think we should return error in these cases.